Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -78,6 +78,11 @@
 #import <Network/Network.h>
 #endif
 
+#if ENABLE(SCREEN_TIME)
+#import <ScreenTime/STWebHistory.h>
```

Comment:
You get this include through `ScreenTimeSoftLink.h` anyway, so it's safe to remove.